ykman-gui/qml/DeviceInfo.qml

```diff
@@ -102,6 +97,15 @@ Item {
                         text: qsTr("Configure...")
                         enabled: isEnabled(modelData.id)
                         visible: parent.features[index].onConfigure !== undefined
+                        focus: true
+                        Keys.onTabPressed: {
+                            var nextButton = btnRepeater.itemAt(index + 1)
+                            if (nextButton.enabled && nextButton.visible) {
+                                nextButton.forceActiveFocus()
+                            } else {
+                                connectionsBtn.forceActiveFocus()
+                            }
+                        }
```

Comment:
Does QML not have something like tab index that would let you do this declaratively?

Reply:
Ok, that makes sense. :+1: